.github/workflows/bump.yml

```diff
@@ -56,19 +56,10 @@ jobs:
       - name: Bump version
         run: ${{ inputs.bump-command }}
```

Comment:
Currently if the semantic commit is only 'build', 'chore', 'docs', 'refactor', 'style' or 'test', `semantic-release` will not bump the version and the **Bump Version** step will fail.

I am tempted to keep this failing, to alert the dev that there is indeed no version bump.

Silencing this error will keep this fact hidden and the implication is, if the dev then executes `ff-rc-to-master` on no version bump the git tag step fail (since the current tag already exists)

Reply:
I've fixed this now.

**version bump**
* it will now skip the commit and push if there was version bump (ci, docs, style, ci)

**git tag**
* will skip the git tag if it already exists

no more errors if we merge / ff to master no version bump changes